pkg/controller/tainteviction/metrics/metrics.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const taintEvictionControllerSubsystem = "taint_eviction_controller"
+
+var (
+	// PodDeletionsTotal counts the number of Pods deleted by TaintEvictionController since its start.
+	PodDeletionsTotal = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Subsystem:      taintEvictionControllerSubsystem,
+			Name:           "pod_deletions_total",
```

Comment:
Any particular reason behind this name, and not the one used in https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/3902-decoupled-taint-manager#beta ?

Reply:
I've opted in for making it consistent with the controller name, taint_eviction_controller. I have actually noticed that the name of the metrics was not updated everywhere in the KEP, I have fixed it in https://github.com/kubernetes/enhancements/pull/4314

Reply:
If this is a new metrics, I'd recommend starting at the `ALPHA` stability level.